svgReact/icons/createReactIconSVGs.ts

```diff
@@ -55,7 +55,7 @@ export const createReactIconSVGs = ({
       .replace(/<\/svg>/gi, '');
 
     const hasSecondaryColor = /secondaryColor/.test(colorVariableContent);
-    const duotoneDefaultColor = isDuotone && hasSecondaryColor ? 'gray800' : 'gray300';
+    const duotoneDefaultColor = isDuotone && hasSecondaryColor ? 'gray800' : 'gray400';
```

Comment:
Interesting, color change again 😆 Imagine we don't have the icon script for this 😆

Reply:
That would be terrible lol